test/overviewSpec.test.js

```diff
@@ -32,13 +47,55 @@ describe('Jest+RTL Workshop', function() {
     })
   })
 
+  // These tests ensure that the form has all of the components and correct number of selections
   it('should not have a size selected on render', () => {
     return waitFor(() => expect(screen.queryByText(/I'll tell/)).toBeInTheDocument())
     .then(() => {
       expect(screen.getByRole('option', { name: '-- select a size --' }).selected).toBe(true)
     })
   })
 
+  it('should display the correct number of options', () => {
+    return waitFor(() => expect(screen.queryByText(/I'll tell/)).toBeInTheDocument())
+    .then(() => {
+      expect(screen.getByTestId('size-options').children.length).toBe(7)
+    });
+  });
+
+  it('should display the correct number of quantities', () => {
+    return waitFor(() => expect(screen.queryByText(/I'll tell/)).toBeInTheDocument())
+    .then(() => {
+      expect(screen.getByTestId('quant-options').children.length).toBe(8)
+    });
+  });
+
+  it('should change the background color to red when clicking add to bag without a size selected', () => {
+    return waitFor(() => expect(screen.queryByText(/I'll tell/)).toBeInTheDocument())
+    .then(() => {
+      return user.click(screen.getByTestId('add-to-bag'))
+    }).then(() => {
+      expect(screen.getByTestId('size-options')).toHaveStyle(`backgroundColor: d107079b`);
+    });
+  });
+
+  it('should change the size with the size select dropdown', () => {
```

Comment:
These tests look great, just make sure they aren't dependent on data that could change with api requests.